task_10/Maxim.Shcherbakan/src/main/java/com/hillel/tasks/StringTasks.java

```diff
@@ -0,0 +1,54 @@
+package com.hillel.tasks;
+
+import java.util.StringJoiner;
+
+public class StringTasks {
+
+  public static void main(String[] args) {
+    StringTasks stringTasks = new StringTasks();
+    int result = stringTasks.binaryToDecimal("1100");
+    System.out.println(result);
+  }
+
+  public String replaceEmail(String email) {
+    return email.replace("@", "[ at ]").replace(".", "[ dot ]");
+  }
+
+  public String shortenWords(String sentence) {
+    String[] array = sentence.trim().split(" ");
+    for (int i = 0; i < array.length; i++) {
+      if (array[i].length() < 4) {
+        continue;
+      }
+      StringJoiner word = new StringJoiner("");
```

Comment:
You don't need to rewrite array
StringJoiner word = new StringJoiner(" "); //space as a separator
//add everything to word, but with only one add at a time (don't put it into array)
then in return just do word.toString()